File: Data Structures & Algorithms/duplicate-integer/submission-2.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        unordered_set<int> thelist;
        for (int n : nums){
            
            if (thelist.count(n)){
                return true;
            }
            thelist.insert(n);

        }
        return false;
    }
};

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-2.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        out = 0
        numsset = set(nums)
        for num in numsset:
            if not num - 1 in numsset:
                count = 0
                while num + count in numsset:
                    count += 1
                out = max(count, out)
        return out

                

